reverse-proxy-traefik/readme.md

```diff
@@ -0,0 +1,142 @@
+# Reverse Proxy Traefik
+
+The package is an alternative reverse proxy nginx, this reverse proxy exposes packages using both domain and subdirectory to host the following services:
+
+| Package  | Hosted                                    |
+|----------|-------------------------------------------|
+| Superset | Domain                                    |
+| Jempi    | Domain                                    |
+| Santempi | Domain                                    |
+| Kibana   | Domain                             |
```

Comment:
This should be subdomains. Make the necessary changes in the respective traefik packages configurations